.github/workflows/e2e-tests-ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - uses: pnpm/action-setup@v2.1.0
         with:
-          version: 6
```

Comment:
I think we should let the [`package.json` determine this](https://github.com/pnpm/action-setup#version):

> Optional when there is a [packageManager field in the package.json](https://nodejs.org/api/corepack.html).

Reply:
This option can go then.

Reply:
Only  ` pnpm/action-setup@v2.2.0` above support that https://github.com/marketplace/actions/setup-pnpm?version=v2.2.0